tests/integration/test_advanced_options.py

```diff
@@ -78,6 +79,165 @@ def test_run_batch_multi(job_controller, monkeypatch) -> None:
             assert ji1.start_time < ji2.end_time
 
 
+def test_run_batch_multi_fail(
+    patch_project,
+    job_controller,
+    daemon_manager,
+    wait_daemon_started,
+    wait_daemon_shutdown,
+    clean_slurm_queue,
+) -> None:
+    from jobflow import Flow
+    from qtoolkit.core.data_objects import CancelStatus
+
+    from jobflow_remote import submit_flow
+    from jobflow_remote.jobs.batch import RemoteBatchManager
+    from jobflow_remote.jobs.state import JobState
+    from jobflow_remote.remote.queue import QueueManager
+    from jobflow_remote.testing import add_sleep
+
+    def submit_jobs(n: int, sleep: int):
+        job_ids = []
+        for _ in range(n):
+            add_j = add_sleep(2, sleep)
+
+            flow = Flow([add_j])
+            submit_flow(flow, worker=worker_name)
```

Comment:
submit_flow can take a Job as an argument, any reason why you create the Flow here ?